frontend/src/components/inventory/CreateInventory.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useEffect } from 'react'
+import {
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  Button,
+  TextField,
+  Autocomplete,
+  Alert,
+  Stack,
+  Paper
+} from '@mui/material'
+import { useInventory } from '../../hooks/useInventory'
+import { useProduct } from '../../hooks/useProduct'
+
+interface Props {
+  open: boolean
+  onClose: () => void
+  onSuccess: () => void
+  binCode: string
+  binID: string
+}
+
+const CreateInventory: React.FC<Props> = ({
+  open,
+  onClose,
+  onSuccess,
+  binCode,
+  binID
+}) => {
+  const defaultQuantity = 1
```

Comment:
const DEFAULT_QUANTITY = 1
```suggestion
  const defaultQuantity = 1
```

Reply:
或者你直接写到下面写1就行